test/integration/admin/forms/businesses/hires_test.rb

```diff
@@ -0,0 +1,33 @@
+require "test_helper"
+
+class Admin::Forms::Businesses::HiresTest < ActionDispatch::IntegrationTest
```

Comment:
I'm not super interested in testing that the content is on the screen. Can you please update this test to mimic full CRUD actions like `test/integration/admin/specialties_test.rb`?

Reply:
fixed in [f09d2c3](https://github.com/joemasilotti/railsdevs.com/pull/818/commits/f09d2c3ec099818723bd5a229087f5d88b5340b5)

Reply:
Do you think testing CRUD actions might fit better in a separate test: `Businesses::HiringInvoiceRequestTest`

Since this test just checks for admin content, we could just remove it in favor of above?

Reply:
I think we can remove the admin content assertions. But I do have a preference towards integration/feature tests over request tests. Even if the only difference is the name of the file!

Reply:
changed in [127ce34](https://github.com/joemasilotti/railsdevs.com/pull/818/commits/127ce34565ec1111de4bef3095c1e7caee6c9c52)

Just in case, Updated naming for `Businesses::HiringInvoice` still incoming in refactor.